include/z64save.h

```diff
@@ -42,142 +42,193 @@ typedef struct {
 } RespawnData; // size = 0x20
 
 typedef struct {
-    /* 0x0000 */ u32 entranceIndex;          // "scene_no"
-    /* 0x0004 */ u8 equippedMask;            // "player_mask"
-    /* 0x0005 */ u8 unk_05;                  // "opening_flag"
+    /* 0x00 */ u32 chest;
+    /* 0x04 */ u32 swch0;
+    /* 0x08 */ u32 swch1;
```

Comment:
The original reason was `switch` is a keyword in C, so that can't be used as a struct member, but currently I have those with a suffix, so it shouldn't make any problem.
I'll rename them